aruba/features/format/jks.feature

```diff
@@ -0,0 +1,169 @@
+    Then the exit status should be 0
+    And "all.jks" should be jks archive with password "123456"
+    Examples:
+       | endpoint  |
+       | test-mode |
+       | TPP       |
+       | Cloud     |
+
+
+  Scenario Outline: where it outputs error when trying to pickup local-generated certificate and output it in JKS format
+    When I enroll random certificate using <endpoint> with -no-prompt -no-pickup
+    And I retrieve the certificate using <endpoint> using the same Pickup ID with -timeout 180 -no-prompt -file all.jks -format jks
+    And it should fail with "key password must be provided"
+    Examples:
+      | endpoint  |
+      | test-mode |
+      | TPP       |
+      | Cloud     |
+
+  Scenario Outline: where it outputs error when trying to enroll certificate in -csr file: mode and output it in JKS format
+    Given I generate random CSR with -no-prompt -csr-file csr.pem -key-file k.pem
+    When I enroll certificate using <endpoint> with -no-prompt -csr file:csr.pem -file all.jks -format jks
+    And it should fail with "The --csr \"file\" option may not be used with the enroll or renew actions when --format is \"jks\""
+    Examples:
+      | endpoint  |
+      | test-mode |
+      | TPP       |
+      | Cloud     |
+
+  Scenario Outline: where it outputs error when trying to enroll certificate in -csr local (by default), -no-pickup and output it in JKS format
+    When I enroll random certificate using <endpoint> with -no-prompt -file all.jks -format jks -no-pickup
+    And it should fail with "The --csr \"local\" option may not be used with the enroll or renew actions when --format is \"jks\" and --no-pickup is specified"
+    Examples:
+      | endpoint  |
+      | test-mode |
+      | TPP       |
+      | Cloud     |
+
+  Scenario Outline: where it outputs error when trying to enroll certificate in -csr local (specified), -no-pickup and output it in JKS format
+    When I enroll random certificate using <endpoint> with -no-prompt -file all.jks -format jks -no-pickup -csr local
+    And it should fail with "The --csr \"local\" option may not be used with the enroll or renew actions when --format is \"jks\" and --no-pickup is specified"
+    Examples:
+      | endpoint  |
+      | test-mode |
+      | TPP       |
+      | Cloud     |
+
+  Scenario Outline: where it pickups up service-generated certificate and outputs it in JKS format
+    When I enroll random certificate using <endpoint> with -no-prompt -no-pickup -csr service
+    And I retrieve the certificate using <endpoint> using the same Pickup ID with -timeout 180 -key-password newPassw0rd! -file all.jks -format jks -jks-alias abc
+    And "all.jks" should be JKS archive with password "newPassw0rd!"
+    Examples:
+      | endpoint  |
+      | test-mode |
+      | TPP       |
+      # | Cloud     | # -csr service is not supported by Cloud
+
+#  Scenario Outline: Pickup JKS with typing pass phrases
```

Comment:
If these code is no longer needed, why not remove it?

Reply:
fixing